addons/auth_totp/models/res_users.py

```diff
@@ -79,6 +89,32 @@ def _totp_try_setting(self, secret, code):
         _logger.info("2FA enable: SUCCESS for %s %r", self, self.login)
         return True
 
+    def get_totp_invite_url(self):
+        return '/web#action=auth_totp.action_activate_two_factor_authentication'
+
+    def totp_invite(self):
+        invite_template = self.env.ref('auth_totp.mail_template_totp_invite', raise_if_not_found=False)
```

Comment:
If you don't raise if it is not found, you should support invite_template being None (aka early return). Or raise if not found, if it is mandatory to have it.